tests/unit/amo/pages/TestUserProfile.js

```diff
@@ -830,23 +783,23 @@ describe(__filename, () => {
       reviews,
       count: DEFAULT_API_PAGE_SIZE + 2,
     });
-    const location = createFakeLocation({ query: { foo: 'bar' } });
 
-    const root = renderUserProfile({ location, params, store });
+    renderUserProfile({ params });
 
-    const paginator = shallow(root.find('.UserProfile-reviews').prop('footer'));
-    expect(paginator.instance()).toBeInstanceOf(Paginate);
-    expect(paginator).toHaveProp('count', DEFAULT_API_PAGE_SIZE + 2);
-    expect(paginator).toHaveProp('currentPage', '1');
```

Comment:
Is there something we can do to assert about this?

Reply:
we don't have all the parameters nicely passed along without shallow rendering, but i figured `Page 1 of 2` was pretty close.